client/src/components/HelpfulBtn.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+
+class HelpfulBtn extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      value: "Was this review ...?",
+      buttonUsefull: true,
+      buttonFunny: true,
+      buttonCool: true,
+      show: false,
+      click: 1
+    };
+    this.handleClickUseful = this.handleClickUseful.bind(this);
+    this.handleClickFunny = this.handleClickFunny.bind(this);
+    this.handleClickCool = this.handleClickCool.bind(this);
+  }
+  handleClickUseful() {
+    this.setState({
+      value: "Thanks for your vote!",
+      buttonUsefull: !this.state.buttonUsefull
+    });
+  }
+  handleClickFunny() {
+    this.setState({
+      value: "Thanks for your vote!",
+      buttonFunny: !this.state.buttonFunny
+    });
+  }
+  handleClickCool() {
+    this.setState({
+      value: "Thanks for your vote!",
+      buttonCool: !this.state.buttonCool,
+      vote: !this.state.vote
+    });
+  }
+  render() {
+    return (
+      <div>
+        <div className="thankyou">{this.state.value}</div>
```

Comment:
This classname thankyou should stay consistent with your others. Either camelCase or snake-case like the rest.

Reply:
@kimbyongjin resolved